tests/examples/opinions/united_states/massappct_u_example.compare.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "case_dates": "2020-12-04",
+    "case_names": "ROBERT FERRIER & others v. COMCAST CORPORATION & others.",
+    "download_urls": "/Disposition/DownloadDisposition/16253?dispositionId=16267",
```

Comment:
Aren't download URLs supposed to be full URLs? If I'm right about that, maybe we should add a test for this across all the sample files as well as fixing this.

Reply:
yes.

Reply:
this is fixed.